docs/tutorial.md

```diff
@@ -394,7 +394,107 @@ What's going on here? Let's step through the code.
 
 (sec_extracting_individuals)=
 
-## Extracting particular SLiM individuals
+## Extracting SLiM individuals
+
+### Extracting a sample of individuals
+
+Often we would like to sample a number of individuals from a population.
+This section provides an example workflow and draws
+attention to some easy-to-miss details for this approach.
+
+Let's start with a nonWF simulation similar to what we've seen previously.
+
+```{literalinclude} sample.slim
+```
+
+```{code-cell}
+%%bash
+slim -s 1 sample.slim
+```
+
+The first, most common method to extract individuals is simply to get all
+those that were alive at a particular time,
+using {meth}`SlimTreeSequence.individuals_alive_at`. For instance, to get
+the list of individual IDs of all those alive at the end of the
+simulation (i.e., zero time units ago), we could do:
+
+```{code-cell}
+orig_ts = pyslim.load("sample.trees")
+alive = orig_ts.individuals_alive_at(0)
+
+print(f"There are {len(alive)} individuals alive in the final generation.")
+```
+
+These are individual IDs. One way to take a sample of living individuals
+and write their SNPs to a VCF is:
+
+```{code-cell}
+np.random.seed(1)
+keep_indivs = np.random.choice(alive, 100, replace=False)
+ts = pyslim.SlimTreeSequence(msprime.mutate(orig_ts, rate=1e-8, random_seed=1))
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=keep_indivs)
+```
+
+
+### Extracting individuals after simplification
+
+In cases where we want to obtain genotypes *after* simplification, we
+need to pay attention to details concerning individual nodes. As
+mentioned earlier, {meth}`.SlimTreeSequence.simplify` takes a list
+of nodes as input:
+
+```{code-cell}
+keep_nodes = []
+for i in keep_indivs:
+    keep_nodes.extend(orig_ts.individual(i).nodes)
+sts = rts.simplify(keep_nodes)
+```
+
+There are often some nodes retained
+after simplification that were not part of our sample. Furthermore, it
+is important to remember that the individual IDs get changed during
+simplification, so we cannot depend on our original list of individual
+IDs. Therefore, we will need to refer
+to the *node* IDs attached to our sample:
+
+```{code-cell}
+ts = pyslim.SlimTreeSequence(msprime.mutate(sts, rate=1e-8, random_seed=1))
+indivlist = []
+for i in ts.individuals_alive_at(0):
+    ind = ts.individual(i)
+    if ts.node(ind.nodes[0]).is_sample():                                                                              
+       indivlist.append(i)
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=indivlist)
+```
+
+:::{note}
+The {meth}`.Node.is_sample` check is a necessary and easy to
+forget step. If this step is omitted, the program will attempt
```

Comment:
We should probably explain why it's necessary a bit more explicitly.

Reply:
Taking another shot at this paragraph:

The Node.is_sample() check is necessary, here. This is important to avoid taking genotypes for any non-sample individuals, i.e. individuals alive at the end of the simulation that had one or more nodes retained in the simplified tree sequence. In fact, we can’t rely on the above error message: sometimes there are alive, non-sample nodes that are not isolated, and these get added to the vcf along with the sample genotypes.

Reply:
How about
```
The {meth}`.Node.is_sample` check is necessary here because there may be individuals alive at the end of the simulation
that are still present in the tree sequence after simplification
despite not being chosen as samples.
This could happen, for instance, if two siblings are in ``keep_indivs`` above but a still-living parent is not.
```
I think explaining the error message should come after the error, not in this `:::{note}`.